lib/fs/basicfs_watch_test.go

```diff
@@ -126,17 +126,17 @@ func TestWatchRename(t *testing.T) {
 	name := "rename"
 
 	old := createTestFile(name, "oldfile")
-	new := "newfile"
+	new_ := "newfile"
```

Comment:
Underscores are ugly; if we don't want to use "new" then something like "newName", "newFile" or whatever, please.

Reply:
Agreed, thx.